src/main/java/frc/robot/subsystems/conveyor/commands/CargoIn.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.subsystems.conveyor.commands;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Ports;
+import frc.robot.Constants;
+import frc.robot.subsystems.conveyor.Conveyor;
+
+public class CargoIn extends CommandBase {
+    private final Conveyor conveyor;
+    private final WPI_TalonSRX motor = new WPI_TalonSRX(Ports.Conveyor.AUX);
+
+    public CargoIn(Conveyor conveyor) {
+        this.conveyor = conveyor;
+        addRequirements(conveyor);
+    }
+
+    @Override
+    public void initialize() {
+        conveyor.setPower(Constants.POWER);
```

Comment:
I'm not sure that would work, you might need to repeatedly call it in `execute()`.